pages/guides/personal-verified-customer/handle-verification-statuses.mdx

```diff
@@ -5,7 +5,7 @@ guide:
   step: 2
 meta:
   title: Customer Verification Statuses | Dwolla API Documentation
-  description: Identify the various statuses of Verified Customer creation, including retry, document and suspended.
+  description: Identify the various statuses of a Personal Verified Customer creation, including retry, document and suspended.
```

Comment:
Let's also add `kba` as an option. 

"Identify the various statuses of a Personal Verified Customer creation, including retry, kba, document and suspended."